blog.md

```diff
@@ -0,0 +1,73 @@
+
+**JavaScript**:
+
+Along with HTML and CSS, JavaScript forms the basis of front-end web development, facilitating the creation of an interactive element. JavaScript is an object oriented text-based programming language used both on the client-side and server-side that allows you to make web pages interactive.Things like animations, video players, interactive maps, and simple in-browser games can be created with JavaScript. To put it simply, where HTML and CSS are languages that give structure and style to web pages, JavaScript gives web pages interactive elements that engage the user.
+
+**PHP**:
+
+PHP is a server side scripting language that is used to develop Static websites or Dynamic websites or Web applications. It runs massive sites such as Facebook and Etsy. It can perform operations like read, write, modify, update and display data from a database. It also provides security and contains safeguards to prevent hacking . Because of its popularity, learning PHP will serve you well if you intend to code for the Web.
+
+**Swift**:
+
+Apple released Swift in June, 2014 as a modern language for developing Mac, iPad, iPhone, Apple Watch, and Apple TV applications. Swift is a general-purpose programming language that has uses ranging from systems programming to mobile and desktop apps and scaling up to cloud services. If you want to enter the world of iOS, Swift is the language with which Apple intends to move forward. 
+
+**Python**:
+
+Python is a general-purpose language and a high level programming language used for everything from server automation to data science. SInce Python is an interpreted programming language, it allows you to run the same code on multiple platforms without recompilation. Python is a great language for beginners, because it is easy to read and understand. Python is used to create web apps like Instagram and help researchers make sense of their data.
+
+**Java**:
+
+Java is a high level object oriented programming language that can be used for anything from web applications to desktop and mobile apps. Code written in Java can be run securely on nearly any platform regardless of the operating system as long as the device has a Java Runtime Environment (JRE) installed. Java has a strong presence among large enterprise applications—like banks, hospitals, and university software. It also powers Android apps, so it’s a good choice for those inclined toward mobile development.
+
```

Comment:
Since the first four and last four languages group together, is it possible to have a table explaining the pros and cons of HTML, JS, PHP, and Swift like the one below for APIs? For example, what are the pros and cons of web development with each language